### PR TITLE
Latex report

### PR DESCRIPTION
**Pull request by alexeyhorkin**
Add a new folder with the initial version of the report (latex files format)

### CODE REVIEW

File: latex_report/Introduction.tex [hunk 1]
```diff
@@ -0,0 +1,16 @@
+\likechapter{Введение}
+
+    Интернет вещей (IoT) объединяет устройства в компьютерную сеть и позволяет им собирать, анализировать, обрабатывать и передавать данные другим объектам через программное обеспечение, приложения или технические устройства.
+
+    
+    Цель проекта "Mini Cooper 2.0" состоит в реализации системы устройств, которые способны передавать информацию о состоянии автомобиля в реальном времени.
```
Comment (alexeyhorkin) [inline latex_report/Introduction.tex]:
```suggestion
    Цель проекта <<Mini Cooper 2.0>> состоит в реализации системы устройств, которые способны передавать информацию о состоянии автомобиля в реальном времени.
```

File: latex_report/Main_part.tex [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+		\centering
+		\includegraphics[scale=0.4]{pictures/8.jpg}
+		\caption{Node-Red архитектура.}
+		\label{fig1}
+	\end{figure}
+	
+
+\section{Thingsboard}
+
+    Для создания проекта нам потребовались следующие активы: "Mini Cooper", "Колесо ЗЛ", "Колесо ЗП", "Колесо ПЛ", "Колесо ПП". В каждом из них указаны взаимоотношения между собой, при этом образуется иерархия.
```
Comment (alexeyhorkin) [inline latex_report/Main_part.tex]:
```suggestion
    Для создания проекта нам потребовались следующие активы: <<Mini Cooper>>, <<Колесо ЗЛ>>, <<Колесо ЗП>>, <<Колесо ПЛ>>, <<Колесо ПП>>. В каждом из них указаны взаимоотношения между собой, при этом образуется иерархия.
```

File: latex_report/Main_part.tex [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+    
+    Были и созданы необходимые устройства: "Давление 1-4", "Температура 1-2", "Спидометр", "GPS".
```
Comment (alexeyhorkin) [inline latex_report/Main_part.tex]:
```suggestion
    Были и созданы необходимые устройства: <<Давление 1-4>>, <<Температура 1-2>>, <<Спидометр>>, <<GPS>>.
```

**Review by alexeyhorkin — COMMENTED** [review]
My fault

**Comment by alexeyhorkin** [comment]
@DimaAmega, please, review =)

**Review by DimaAmega — APPROVED** [review]
nice job 👌😁

**Pull request merged**